src/script.js

```diff
@@ -0,0 +1,37 @@
+let myInput = document.getElementById("pswd1");
+let repeatInput = document.getElementById("pswd2")
+let letter = document.getElementById("letter");
+let capital = document.getElementById("capital");
+let number = document.getElementById("special sign");
+let length = document.getElementById("length");
+
+function checkPass() {
+    // Validate lowercase letters
+    let lowerCaseLetters = /[a-z]/;
+    if (myInput.value.match(lowerCaseLetters)) {
+        letter.remove();
```

Comment:
Jak wstawię małą literę to komunikat o braku małej litery znika, a jak ją potem usunę to komunikat się nie pojawia ponownie, a powinien, analogicznie dla pozostałych komunikatów